src/ThunderTestTrait.php

```diff
@@ -83,4 +86,66 @@ protected function logWithRole($role) {
     $this->drupalLogin($editor);
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  protected function tearDown() {
+    /** @var \Drupal\Core\Database\Query\SelectInterface $query */
+    $query = \Drupal::database()->select('watchdog', 'w')
+      ->fields('w');
+    $andGroup = $query->andConditionGroup()
+      ->condition('severity', 5, '<')
+      ->condition('type', 'php');
+    $group = $query->orConditionGroup()
+      ->condition('severity', 4, '<')
+      ->condition($andGroup);
+    $query = $query->condition($group);
```

Comment:
Is it possible to group the messages in the query, to filter out douplicates?
This would also prevent the ugly print_r(array_unique($errors), TRUE) line